### PR TITLE
Add `elementor-image` & `elementor-text-editor` classes to DOM optimization experiment [ED-2095]

### PR DESCRIPTION
**Pull request by arielk**

### CODE REVIEW

File: assets/dev/scss/frontend/widgets/image.scss [hunk 1]
```diff
@@ -4,27 +4,16 @@
 .elementor-widget-image {
 	text-align: center;
 
-	.elementor-image {
+	a {
```
Comment (kobizz) [inline assets/dev/scss/frontend/widgets/image.scss:7]: This softer selector might allow other selectors to override it. Could you keep it like before?
```
> a,
figure > a
```

**Review by kobizz — CHANGES_REQUESTED** [review]

File: includes/widgets/image.php [hunk 2]
```diff
@@ -777,8 +779,10 @@ protected function content_template() {
 				link_url = settings.image.url;
 			}
 
-			#><div class="elementor-image{{ settings.shape ? ' elementor-image-shape-' + settings.shape : '' }}"><#
-			var imgClass = '';
+			<?php if ( ! Plugin::$instance->experiments->is_feature_active( 'e_dom_optimization' ) ) { ?>
+				#><div class="elementor-image{{ settings.shape ? ' elementor-image-shape-' + settings.shape : '' }}"><#
+				var imgClass = '';
```
Comment (kobizz) [inline includes/widgets/image.php:784]: This line should be outside of the condition

**Review by kobizz — CHANGES_REQUESTED** [review]

File: includes/widgets/image.php [hunk 3]
```diff
@@ -777,8 +779,10 @@ protected function content_template() {
...
+			<?php } ?>
```
Comment (kobizz) [inline includes/widgets/image.php:785]:
```suggestion
			<?php } ?>
				var imgClass = '';
```

**Review by kobizz — CHANGES_REQUESTED** [review]

File: includes/widgets/image.php (hunk 3 above)
Comment (arielk) [reply includes/widgets/image.php]: Fixed: https://github.com/elementor/elementor/pull/13915/commits/c40c6d9c151cd3c73948306b667b9a0e8ef3c689

**Review by kobizz — APPROVED** [review]

**Pull request merged**